engine/js/ng/sites/sections/entries/section-entries.reducer.js

```diff
@@ -14,35 +14,137 @@
       switch (action.type) {
 
         case ActionTypes.SET_STATE:
-          return Immutable.fromJS(action.state.section_entries);
+          return Immutable.fromJS(action.state.sectionEntries);
```

Comment:
didn't we fix the `SET_STATE` action already, so that each reducer would have it's own set initial state action?